CMakeLists.txt

```
@@ -173,7 +178,7 @@ set(LUA_FOUND true)
 message("CMake build ${CMAKE_BUILD_TYPE}, CFLAGS=${CMAKE_C_FLAGS}, CXXFLAGS=${CMAKE_CXX_FLAGS}")
 
 add_definitions(-D_MT -D_CPPUNWIND -DPURE_DYNAMIC_CAST -DDECLARE_SPECIALIZATION -DM_NOSTDCONTAINERS_EXT)
-add_definitions(-DLUABIND_DYNAMIC_LINK) #self-explanatory
+add_definitions(-DLUABIND_DYNAMIC_LINK)
```

Don't unnecessary diffs.
```suggestion
add_definitions(-DLUABIND_DYNAMIC_LINK) #self-explanatory
```